Clarifications on downgrade and integrations (#501)
DataDog/ansible-datadog: README.md

| `datadog_zypper_gpgkey_e09422b3`            | Override the default URL to the Datadog `zypper` key used to verify Agent v6.14+ packages (key ID `E09422B3`).                                                                                                                                                                                                                                                                                                                                    |
| `datadog_zypper_gpgkey_e09422b3_sha256sum`  | Override the default checksum of the `datadog_zypper_gpgkey_e09422b3` key.                                                                                                                                                                                                                                                                                                                                                                        |

> | `datadog_agent_allow_downgrade`             | Set to `yes` to allow Agent downgrade (use with caution, see `defaults/main.yml` for details). **Note**: Downgrades are not supported on Windows platforms. On Centos it only works with Ansible 2.4+.|

Reviewer: ```suggestion
| `datadog_agent_allow_downgrade`             | Set to `yes` to allow Agent downgrade (use with caution, see `defaults/main.yml` for details). **Note**: Downgrades are not supported on Windows platforms. On CentOS, Agent downgrade only works with Ansible 2.4+.|
```

Reviewer 2: The role only supports Ansible 2.6+, so I'd remove the lines about Ansible 2.4+.

### Integrations

To configure a Datadog integration (check), add an entry to the `datadog_checks` section. The first level key is the name of the check, and the value is the YAML payload to write the configuration file. Examples are provided below. 

> To install or remove an integration, please refer to the `datadog_integrations` [paragraph][22]

Reviewer: ```suggestion
To install or remove an integration, refer to the `datadog_integrations` [paragraph][22]
```

To upgrade from Agent v6 to v7, use `datadog_agent_major_version: 7` to install the latest version or set `datadog_agent_version` to a specific version of Agent v7. Use similar logic to upgrade from Agent v5 to v6.

> #### Integrations installation

Reviewer: ```suggestion
#### Integration installation
```

**Available for Agent v6.8+**

> Use the `datadog_integration` resource to install a specific version of a Datadog integration. Keep in mind, the Agent comes with all the integrations ([core integrations][19]) already installed. This command is useful for upgrading a specific integration without upgrading the whole Agent. For more details, see [Integration Management][4].

Reviewer: ```suggestion
Use the `datadog_integration` resource to install a specific version of a Datadog integration. Keep in mind, the Agent comes with the [core integrations][19] already installed. This command is useful for upgrading a specific integration without upgrading the whole Agent. For more details, see [integration management][4].
```

> If you want to configure an integration, please refer to the `datadog_checks` [paragraph][21]

Reviewer: ```suggestion
If you want to configure an integration, refer to the `datadog_checks` [paragraph][21]
```

Available actions:

- `install`: Installs a specific version of the integration.
- `remove`: Removes an integration.

##### Third party integrations

> [Datadog Community][20] and [Datadog Marketplace][15] integrations can be installed with the `datadog_integration` resource. **Note**: These integrations are considered to be "third party" and thus need `third_party: true` to be set - see the example below.

Reviewer: ```suggestion
[Datadog community][20] and [Datadog Marketplace][15] integrations can be installed with the `datadog_integration` resource. **Note**: These integrations are considered to be "third party" and thus need `third_party: true` to be set—see the example below.
```

**Notes:**

> - Downgrades are not supported for Windows platforms. On CentOS it requires Ansible 2.4+

Reviewer: ```suggestion
- Downgrades are not supported for Windows platforms. On CentOS, downgrades require Ansible 2.4+
```
